src/store/modules/project_params.js

```diff
@@ -172,7 +175,14 @@ const mutations = {
   start_date (state, val) { state.start_date = val },
   long_stack (state, val) { state.long_stack = val },
   smart_stack (state, val) { state.smart_stack = val },
-  defocus (state, val) { state.defocus = val }
+  defocus (state, val) { state.defocus = val },
+  draft_project_params (state, val) { state.draft_project_params = val }, // project params that have saved off in saveProjectDraft
+  rewrite_state (state, val) {
+    // loop through and rewrite the state
+    for (const stateItem in val) {
+      Vue.set(state, stateItem, val[stateItem])
```

Comment:
In order to restore the state from the draft_project_params, we need to loop through every attribute in the draft params and set the corresponding state variable.